flowetl/etl/tests/test_move_file_and_record_ingestion_state__callable.py

```diff
@@ -0,0 +1,48 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+# -*- coding: utf-8 -*-
+from unittest.mock import patch, MagicMock
+from pathlib import Path
+
+from etl.production_task_callables import move_file_and_record_ingestion_state__callable
+
+
+def test_move_file_and_record_ingestion_state__callable(tmpdir, create_fake_dag_run):
+
+    from_dir = tmpdir.mkdir("from_dir")
+    to_dir = tmpdir.mkdir("to_dir")
+
+    file_name = "file_to_move"
+    file = from_dir.join(file_name)
+
+    file_contents = """
+    Some contents in
+    a file..
```

Comment:
```suggestion
    a file.
```

Reply:
Nooooooooooooooooo I want an ellipsis of some form in the bleeding code...

Reply:
But an ellipsis has three dots!
```suggestion
    a file...
```

Reply:
That's why I said "of some form"...

Reply:
Lol!